packages/app/src/components/PageEditor/EmojiPicker.tsx

```diff
@@ -1,55 +1,35 @@
-import React, { FC, useRef, useEffect } from 'react';
-
+import React, { FC } from 'react';
 import { Picker } from 'emoji-mart';
 import i18n from 'i18next';
-
+import { Modal } from 'reactstrap';
 import { isDarkMode } from '~/client/util/color-scheme';
-
 import EmojiPickerHelper from './EmojiPickerHelper';
 
 type Props = {
   onClose: () => void,
   emojiSearchText: string,
   editor: any
-  emojiPickerHelper: EmojiPickerHelper
+  emojiPickerHelper: EmojiPickerHelper,
+  isOpen: boolean
 }
 
 const EmojiPicker: FC<Props> = (props: Props) => {
 
   const {
-    onClose, emojiSearchText, emojiPickerHelper,
+    onClose, emojiSearchText, emojiPickerHelper, isOpen,
   } = props;
 
-  const emojiPickerContainer = useRef<HTMLDivElement>(null);
-
-  useEffect(() => {
-
-    if (emojiSearchText != null) {
-      // Get input element of emoji picker search
-      const input = document.querySelector('[id^="emoji-mart-search"]') as HTMLInputElement;
+  // Set search emoji input and trigger search
+  const searchEmoji = () => {
+    if (emojiSearchText !== null) {
+      const input = window.document.querySelector('[id^="emoji-mart-search"]') as HTMLInputElement;
```

Comment:
It seems better to use `window.document.getElementById()` 👀

Reply:
The Id of `emoji-mart-search-` is dinamically generated . The id will always change after we re-open the EmojiPicker

Reply:
I got it! Thanks for the description. 🥇

Reply:
I got it! Thanks for the description. 🥇